data/get_html_IO.py

```diff
@@ -8,9 +8,11 @@
 3. Run this script
 
 Outputs:
-"inputs.txt": html code documenting the inputs
-"measurements.txt": html code documenting the outputs
-
+"inputs.html": html code documenting the inputs
+"measurements.html": html code documenting the outputs
+"forecast_points.html" html code documenting the forecasts
+"inputs_measurement_forecasts.html" html code documenting inputs,outputs and 
```

Comment:
I don't think we need the individual .html files if the combined one is created, right?  Suggest removing those outputs if not.  Also, please rename the output "inputs_measurement**s**_forecasts.html" for consistency.

Reply:
@dhblum thank you for the comment. I removed the portion of the code that generates the single files.